### ISSUE TITLE
Make posting Research Notes more clear & inviting, and the central feature of the website

### ISSUE THREAD

**Issue by jywarren** [jywarren/plots#161]
Based on watching people using the site, and my own usage, I'd like to make a specific push for emphasizing Research note posting as the _main_ thing people are invited to do on the PLOTS website. We'd accomplish this in a few different ways:
- make the posting button more obvious (color?)
- link to explanatory page (see Chris's below)
- offer a very brief video explaining why research notes are important and how to post one
- prompt new users (ones without any notes) to post them with helpful info (see prev.)
- more explicitly link posting research notes to _actually open sourcing your work_

There's some overlap with specific other issues, like:
- Simplify the note posting form: #53
- Add/improve WYSIWYG rich text editor: #117
- Make image uploading easier: #24

I also want to attach some thoughts Chris Fastie posted on the topic, which helped to define the problem and suggest solutions:

> I think research notes work really well the way they are.  It’s good to have some constraints so they all have a similar style, and there is enough functionality to add rich media and formatting if desired.  But it took me a while to learn what could be included and how to do it.  At first I completely missed the “main” photo, and it took me a few tries to figure out how the gallery worked. My first attempt to embed a video and MapKnitter map seemed to fail because of the flash bug so I deleted the whole note.  Fortunately, I was willing to email you and learn that iframes will appear when the note is saved and the page refreshed.  Many others may have given up instead at that point. I almost did.
> So the problem with posting notes is mostly that the process is a bit obscure and that there is absolutely NO GUIDANCE.  A short list of tips is mostly all it needs:

Link to Chris's list of tips/guidance, which I've added to a bit: http://publiclaboratory.org/wiki/posting-research

> Not having something like these six tips conspicuously displayable somewhere seems like a strong message that you don’t really want lots of people posting research notes. 
> Another obstacle to people posting notes is that most of them probably don’t know that they can, and many others don’t know that they should.  This has to be made explicit somewhere.  As does the relationship between notes and the discussion forum.  It’s not very hard to make it all explicit.

And a link to the bug about embedded flash/javascript failing to show until you refresh the page: #162

**Comment by mathewlippincott** [comment]
I totally agree with Chris's thoughts, especially from the perspective of someone who already is used to posting web content-- I have youtube,vimeo, flickr pro accounts, my own web hosting, etc-- and our research notes play well with these.  I also like the ability to use quick wiki syntax for links and stuff like that.  

the push for WYSIWYG is because most people will never learn wiki syntax, and will ignore the research note functionality until WYSIWYG is implemented.  We certainly need a clear link explaining how to use it, but I'd rather put my energy towards bugging jeff to write a WYSIWYG feature ;)  

My pessimism towards pushing the existing research note system is because all these activities--  the explanitory page, the video, the prompting of users, will 
1) be counter-productive- introducing novice users to a system they won't find friendly (non-wysiwyg), potentially putting them off of trying a second time once we make it user-friendly.
2) have to be re-done once wysiwyg is implemented.

**Comment by jywarren** [comment]
I dunno - i think maybe there are 2 threads here: i just recorded a short movie (it could be shorter) quickly pitching WHY to post, and quickly showing HOW. There was almost no time spent on actually crafting text, styling it, or embedding images. I think this "minimal" way of posting is fine, and would like to make it more clear to people in a variety of ways. I think it'd be nice if people basically thought of research notes as a lightweight and easy way to just dump content and show others what they're doing.

The WYSIWYG issue (#117) is a bit of a different use case, and I _also_ agree with it. But because you can simply post content without any styling at all -- ignoring Markdown or rich text entirely -- I think it may be less about first-time users and more about people who want to present detailed and customized documentation. I don't necessarily think WYSWYG  would mean we'd have to start over or anything.

I have more to say about WYSIWYG but i'm moving over to that thread: #117

**Comment by mathewlippincott** [comment]
Ignore this comment and see WYSIWYG thread #117

you're right about the minimal posting method being fine for many things-- but perceptions are as important as potentials.  Just because minimally formatted notes have the potential to communicate 90% of what people have to say doesn't mean they'll be perceived that way.

No one wants to be the novice user making content they perceive to be poorly formatted vis a vis the existing content on our site, or have to admit to other users they don't know how to do something.  I think many people just won't start using our system until they know how to make their content look like everyone else's.  

We have set a certain bar with formatting by using wiki markup and embedded images, and until others can get to that bar, it will constitute a block to usability, because we've established an _avoidable_ division between technical and non-technical users.

**Comment by mathewlippincott** [comment]
The red add research note button is stellar. Noticed it immediately.

**Comment by rjcorwin** [comment]
@jywarren Awesome writeup and that's some great feedback from Chris!  The new red did jump out at me as well but I wonder if that's because I've seen the site so many times that it seemed unusual.  To up the in-your-face factor we could also make the button follow the user around...  

Also another thing to think about is what's on the landing page.  If there is a use case you want a user to follow on a site, I think the best way to let them know that is to spell it out for them on the landing page and above the fold.
1. This is what you could do.
2. This is why you want to do that.
3. Here are other people like you having success doing that.
4. Here is where you can do that. Get going!

Make all of those statements for research notes above the fold + images + animation and I think we'd see a big difference in the number of people who know they can contribute.

Related -> https://github.com/jywarren/plots/issues/49

**Comment by jywarren** [comment]
Lets break this up into a few things we can do, then do them one by one and see how we like each. Then maybe leave all of them up. How about:
- redesign posting form, radically simplify it and link to help (wiki) pages as in #53
- create alert-box-like "prompts" for new users, linking to a short video and a list of tips
- consider WYSIWYG features in #117
- improve/simplify image uploading (#24), which may depend on the WYSIWYG we install, so probably has to come after...

Generally let's follow the principle that advanced features could be distracting/confusing, and should be slightly obscured, but exposed by a "More options" link?

**Comment by jywarren** [comment]
i just posted a proposed much simpler note posting form in #53 if you're not following that thread

**Comment by mathewlippincott** [comment]
OK-- I'll remember to post my next note using the form.  

One note-- just got an e-mail from a user wondering where to find the "note I was working on yesterday" but didn't save.  Can we add a "save" function that is separate from "post"?
